module/EntraBeta/AdditionalFunctions/Set-EntraBetaDirSyncEnabled.ps1

```diff
@@ -0,0 +1,51 @@
+# ------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+# ------------------------------------------------------------------------------
+
+function Set-EntraBetaDirSyncEnabled {
+    [CmdletBinding(DefaultParameterSetName = 'All')]
+    param (
+        [Parameter(ParameterSetName = "All", ValueFromPipelineByPropertyName = $true, Mandatory = $true)][System.Boolean] $EnableDirsync,
+        [Parameter(ParameterSetName = "All", ValueFromPipelineByPropertyName = $true)][System.Guid] $TenantId,
+        [switch] $Force
+    )
+
+    PROCESS {
+        $params = @{}
+        $customHeaders = New-EntraBetaCustomHeaders -Command $MyInvocation.MyCommand
+        if ($PSBoundParameters.ContainsKey("Verbose")) {
```

Comment:
Since we are calling a Graph cmdlet, we can add all common parameters here

Reply:
Updated code!